### ISSUE TITLE
Local Wireless doesn't work

### ISSUE THREAD

**Issue by butterfly2706** [rafaelvcaetano/melonDS-android#1230]
Local wireless communication is still not working. The last version it was working properly is ver1.70.

I'm testing via 2 instances of melonds using an app cloner

**Comment by shabbirmahmud** [comment]
Wah still want wfc for ds??

**Comment by STEVE-916-create** [comment]
Local wireless communication is not possible in DS to Android since the DS has a custom Wireless chip able to send custom frames/packets to nearby DSes.

**Comment by butterfly2706** [comment]
> Local wireless communication is not possible in DS to Android since the DS has a custom Wireless chip able to send custom frames/packets to nearby DSes.

Except that it did used to.

This is connecting two different Pokemon Games in a single phone via Local Wireless, using v1.70

![Screenshot_2023-11-03-18-11-22-46_29e3ed2390198a05a01ed02ad7fbd5de](https://github.com/rafaelvcaetano/melonDS-android/assets/80578498/aa32e06e-34df-400a-af4f-74fc700f39c4)

**Comment by ashgx** [comment]
Regarding local multiplayer on the same device, it's no longer possible as of the last release. Since melonDS favoured IPC communication for local multiplayer, I dropped the socket implementatio as well since it was very unreliable. The IPC approach shows some potential from my experiments, but it's not usable in the current state. So, there's no way to use local multiplayer on Android. 
What the author said.

**Comment by ashgx** [comment]
![112](https://github.com/rafaelvcaetano/melonDS-android/assets/81661156/7207f619-9735-4294-ba41-d82e112a9960)

**Comment by ashgx** [comment]
china？

**Comment by butterfly2706** [comment]
I see, well then since I primarily just use Local , I guess I have to stick with v1.7

**Comment by rafaelvcaetano** [comment]
Yes, as stated in the quote posted above, local multiplayer has been unavailable since 1.8. Since the PC version dropped it's old implementation and it was actually very hacky on Android, I decided to drop it as well and start working on using the same approach the PC version uses to proper local multiplayer. However, it's still not usable. So to keep using local multiplayer, you will need to keep using the Beta 1.7 versions.

**Comment by ShipperNessity** [comment]
Does that mean, even the newly released 1.9.0 doesn't have the wireless connection feature as well?

**Comment by GamesPlays25** [comment]
ZeroTier One?
